### PR TITLE
PLT-662: Establish Athena DataSets 

### PR DESCRIPTION
**Pull request by timothy-ellis-ky**
## 🎫 Ticket

https://jira.cms.gov/browse/PLT-662

## 🛠 Changes

- Update commentary and definition for Athena Saved Queries (create or replace view)
- Update IAM permissions to include QuickSight actions where appropriate, including S3 CMK access

## ℹ️ Context

These changes were made to enable manual creation of QuickSight data sets, Analyses, and Dashboards

## 🧪 Validation

See Screen shots in Jira Ticket

### CODE REVIEW

File: terraform/services/quicksight/main.tf [hunk 1]
```diff
@@ -45,4 +48,10 @@ module "dpc_insights_data" {
 module "dpc_insights_athena" {
   source = "../../modules/bucket"
   name   = local.dpc_athena_s3_name
+}
+
+resource "aws_s3_object" "folder" {
+  bucket       = module.dpc_insights_athena.id
+  content_type = "application/x-directory"
+  key          = local.dpc_athena_results_folder_key
```
Comment (gsf) [inline terraform/services/quicksight/main.tf:56]: I'm a little unclear as to why this resource is needed. Is there an error if the output_location is supplied without this?

Comment (timothy-ellis-ky) [reply terraform/services/quicksight/main.tf:56]: TF definitely argued when I didn't have a specific Result_Configuration folder of this object type for the Athena Workgroup resource; I pulled over the comparable config from BFD-Insights

Comment (gsf) [reply terraform/services/quicksight/main.tf:56]: Still seems odd to me. I don't see any docs about the need to define an x-directory object in S3 for Athena. But it's fine to leave it in for now.

**Review by gsf — APPROVED** [review]
Great work

**Pull request merged**